tests/regressiontests/make/tests/tests-base.m4

```diff
@@ -146,6 +146,14 @@ ADD_TEST_BASH([test-simple], [test_simple_body
 	$< pos -o 'uf ta' | grep -q 'OPT_S=uf ta,POS_S=pos,'
 ])
 
+ADD_TEST_BASH([test-simple], [test_simple_body
+	$< pos -o 'uf ta' --print-optionals | grep -q 'OPT_S=uf ta,POS_S=pos,LA=,_supplied_arg_prefix=1,_supplied_arg_print_optionals=1,_supplied_arg_la=x'
+])
+
+ADD_TEST_BASH([test-simple], [test_simple_body
+	$< pos -o 'uf ta' --print-optionals --la x | grep -q 'OPT_S=uf ta,POS_S=pos,LA=x,_supplied_arg_prefix=1,_supplied_arg_print_optionals=1,_supplied_arg_la=1'
+])
+
```

Comment:
And you can take this out - extending that test above enough.
It tests more things at once, but that's not a problem.